src/internal/storage/chunk/option.go

```diff
@@ -65,3 +69,19 @@ func WithNoUpload() WriterOption {
 		w.noUpload = true
 	}
 }
+
+// StorageOptions returns the chunk storage options for the config.
+func StorageOptions(conf *serviceenv.Configuration) ([]StorageOption, error) {
```

Comment:
Moved from `serviceenv` to avoid a circular reference.

Reply:
Oh cool, these options survived.